sdk/core/core-lro/test/lro.spec.ts

```diff
@@ -1077,6 +1077,47 @@ matrix(
               assert.equal(result.id, "100");
             });
 
+            it("should handle patchAsyncNoLocationHeader", async () => {
+              const resourceLocationPath = `/patchasyncnolocationheader/succeeded`;
+              const initialResourcePath = resourceLocationPath;
+              const pollingPath = `/patchasyncnolocationheader/operationresults/123`;
+              const result = await runLro({
+                routes: [
+                  {
+                    method: "PATCH",
+                    status: 202,
+                    path: initialResourcePath,
+                    headers: {
+                      [headerName]: pollingPath,
+                    },
+                  },
+                  {
+                    method: "GET",
+                    path: pollingPath,
+                    status: 200,
+                    body: `{ "status": "InProgress"}`,
+                    headers: {
+                      "Azure-AsyncOperation": pollingPath,
+                    },
+                  },
+                  {
+                    method: "GET",
+                    path: pollingPath,
+                    status: 200,
+                    body: `{ "status": "Succeeded"}`,
+                  },
+                  {
+                    method: "GET",
+                    path: resourceLocationPath,
+                    status: 200,
+                    body: `{ "name": "sku" , "id": "100" }`,
+                  },
+                ],
+              });
+              assert.equal(result.name, "sku");
+              assert.equal(result.id, "100");
+            });
+
```

Comment:
Can you add another unit test that corresponds to the problematic operation you're dealing with? The status code of the initial response to be 201 and the headers don't contain a `Location` one.

Reply:
Sure let me add it later.